File: CollaborationFront/WebContent/Job/jobservice.js
/**
 * JOB SERVICE
 */
app.factory('JobService',function($http){
	
	var jobService={}
	var BASE_URL="http://localhost/CollaborationMiddle"
	jobService.addjob=function(job)
	{
	return $http.post(BASE_URL+"/addjob",job)
	}
	jobService.getalljobs=function()
	{
	return $http.get(BASE_URL+"/getalljobs")
	}
	jobService.getJobDetails=function(jobId)
	{
		return $http.get(BASE_URL+"/getjobbyid/"+jobId)
	}  
	
	return jobService;
})

File: CollaborationBack/src/main/java/com/collab/CollaborationBack/Service/JobService.java
package com.collab.CollaborationBack.Service;

import java.util.List;

import com.collab.CollaborationBack.model.Job;

public interface JobService {

	public boolean addjob(Job job);
	public boolean updatejob(Job job);
	public boolean deletejob(int jobId);
	List<Job> getalljob();
	Job getjobById(int jobId);
}
